src/System.Diagnostics.StackTrace/tests/Configurations.props

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      uapaot-Windows_NT;
```

Comment:
no need to add all of these. It looks by checking your .csproj that the only special case is netfx, so I would have it like:
```xml
<BuildConfigurations>
  netstandard;
  netfx;
</BuildConfigurations>
```
By doing that, then the netstandard configuration will be built everywhere except on netfx, which has a specific configuration. If you are interested on how our configurations work and how do we pick each one let me know and I can gladly help :smile: